snippets/base/templates/base/includes/snippet_as.js

```diff
@@ -518,6 +518,9 @@ function addToBlockList(snippetID) {
     if (blockList.indexOf(snippetID) === -1) {
         blockList = [snippetID].concat(blockList);
         gSnippetsMap.set('blockList', blockList);
+        if (gSnippetsMap.blockSnippet) {
+          gSnippetsMap.blockSnippet(snippetID);
+        }
```

Comment:
this should be an else `gSnippetsMap.set('blockList', blockList);` instead of that being above;
also, can you add a comment with a link to the bug number explaining this? thanks!